app/core/CONFIG.md

```diff
@@ -62,4 +62,14 @@ Set to `true` if you want your chapter downloads to be compressed into a zip fol
 - `zipType`
 
 Valid options are `zip` or `cbz`. This is ignored if `asZip` is set to `false`. Any other empty/invalid option will
-default to `zip`.
\ No newline at end of file
+default to `zip`.
+
+
+### Guest Login
+
+- `guestLogin`
```

Comment:
i personally think something like `guestMode` rolls off the tongue smoother!

Reply:
Updated!